### PR TITLE
Edit finish-response and change generated BCSet name

### PR DESCRIPTION
**Pull request by jomrazek**

### CODE REVIEW

File: bc-backend/src/main/java/org/jboss/da/bc/backend/impl/FinalizerImpl.java [hunk 1]
```diff
@@ -43,16 +43,17 @@
...
-    @Asynchronous
-    public void createBCs(String name, String productVersion, ProjectHiearchy toplevelBc,
+    public Integer createBCs(String name, String productVersion, ProjectHiearchy toplevelBc,
             String bcSetName) {
         try {
             Set<Integer> ids = create(toplevelBc);
             int productVersionId = bcSetGenerator.createProduct(name, productVersion);
             bcSetGenerator.createBCSet(bcSetName, productVersionId, new ArrayList(ids));
+            return productVersionId;
         } catch (Exception ex) {
```
Comment (janinko) [inline bc-backend/src/main/java/org/jboss/da/bc/backend/impl/FinalizerImpl.java]: I think we should throw the exception now that the method is not asynchronous.

File: bc-backend/src/main/java/org/jboss/da/bc/backend/impl/FinalizerImpl.java [hunk 2]
```diff
@@ -43,16 +43,17 @@
...
             log.error("Failed to finalize produc import process", ex);
         }
+        return null;
```
Comment (janinko) [inline bc-backend/src/main/java/org/jboss/da/bc/backend/impl/FinalizerImpl.java]: I think we should throw the exception now that the method is not asynchronous.

Comment (jbartece) [inline bc-backend/src/main/java/org/jboss/da/bc/backend/impl/FinalizerImpl.java]: OK. I'll fix it

**Comment by jbartece** [comment]
Added fix in https://github.com/project-ncl/dependency-analysis/pull/175

**Pull request closed without merging**